### PR TITLE
Update HD radio parameters

### LINKED ISSUE #83: [SDL 0160] Remote Control Radio Parameter Update
Proposal: [Remote Control Radio Parameter Update](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0160-rc-radio-parameter-update.md)

> This proposal adds a new parameter `hdRadioEnable` to `RadioControlData` data structure, adds new parameters `hdRadioEnableAvailable` and `siriusxmRadioAvailable` to `RadioControlCapabilities` data structure, and changes the maximum number of HD radio sub-channels from 3 to 7.

Review: https://github.com/smartdevicelink/sdl_evolution/issues/464

### PR DESCRIPTION
**Pull request by jacobkeeler**
Resolves #83

### CODE REVIEW

**Comment by jacobkeeler** [comment]
Note that this is targeting `feature/new_rc_modules_and_parameters`, this will need to be changed before merging

File: MOBILE_API.xml [hunk 1]
```diff
@@ -2670,11 +2670,20 @@
         </param>
         <param name="rdsData" type="RdsData" mandatory="false">
         </param>
-        <param name="availableHDs" type="Integer" minvalue="1" maxvalue="3" mandatory="false">
+        <param name="hdRadioEnable" type="Boolean" mandatory="false"  since="5.0">
+            <description> True if the hd radio is on, false is the radio is off</description>
```
Comment (theresalech) [inline MOBILE_API.xml]: Should this say `false if the radio is off` (instead of `false is the radio is off`)?

File: README.md [hunk 2]
```diff
@@ -2100,6 +2100,7 @@ Contains information about this system's video streaming capabilities.
 |`frequencyFraction`|Integer|False|The fractional part of the frequency for 101.7 is 7|
 |`band`|RadioBand|False||
 |`rdsData`|RdsData|False||
+|`hdRadioEnable`|Boolean|False|True if the hd radio is on, false is the radio is off|
```
Comment (theresalech) [inline README.md]: Should this say `false if the radio is off` (instead of `false is the radio is off`)?

File: MOBILE_API.xml (hunk 1 above)
Comment (theresalech) [inline MOBILE_API.xml]: Should this say false if the radio is off (instead of false is the radio is off)?

**Review by theresalech — CHANGES_REQUESTED** [review]
@jacobkeeler two questions for your review.  Can you please advise and update accordingly?  Thanks!

**Comment by jacobkeeler** [comment]
@theresalech comments addressed

**Review by theresalech — APPROVED** [review]

**Review by joeygrover — APPROVED** [review]

**Pull request merged**